home/components/SnackListItem.tsx

```diff
@@ -29,11 +30,45 @@ function SnackListItem({ url, subtitle, ...props }: Props) {
   return (
     <ListItem
       subtitle={normalizeDescription(subtitle)}
+      rightContent={
+        isDraft ? (
+          <View style={styles.rightContentContainer}>
+            <View style={styles.draftContainer}>
+              <Text style={styles.draftText} numberOfLines={1} ellipsizeMode="tail">
+                Draft
+              </Text>
+            </View>
+          </View>
```

Comment:
Maybe extract this into separate component? 🤔 
```suggestion
          <DraftLabel />
```

Reply:
Great suggestion, I've added a `<Badge>` component and updated the code accordingly 👍